### PR TITLE
Test Cases updated

### PR DESCRIPTION
**Pull request by estebanpalomares**
Updated and executed, all passed

### CODE REVIEW

**Review by wiensamo — APPROVED** [review]
big advance.

File: src/main/java/com/gworks/ui/services/FillFormService.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                loginPasswordBox.sendKeys("");
+                break;
+
+            default:
+                Assert.fail("The case "+testCase+" does not exist.");
+        }
+    }
+
+    @Override
+    public void selectCityInCityPage(WebDriver webDriver) {
```
Comment (luz-angelica) [inline src/main/java/com/gworks/ui/services/FillFormService.java:52]: add this select in a new method is a better way to do 🥇 
Thanks

File: src/test/java/com/gworks/stepDefinitions/ui/CityPageLogInStepDef.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    public void theUserUsesACredentialsInFrontDeskPage(String testCase) throws InterruptedException {
+        TimeUnit.SECONDS.sleep(4);
+        this.fillFormController.fillLoginPage(this.webDriver,testCase);
+        this.fillFormController.clickLoginButton(this.webDriver);
+        TimeUnit.SECONDS.sleep(4);
+        this.fillFormController.selectCityInCityPage(this.webDriver,testCase);
+        this.fillFormController.clickLoginButton(this.webDriver);
+    }
+
+    @Then("I am logged in FrontDesk")
```
Comment (luz-angelica) [inline src/test/java/com/gworks/stepDefinitions/ui/CityPageLogInStepDef.java]: chance this Then on third person please

File: src/test/java/com/gworks/stepDefinitions/ui/CityPageLogInStepDef.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    }
+
+    @When("The user uses a fail {string} credentials in FrontDesk page")
+    public void theUserUsesAFailCredentialsInFrontDeskPage(String testCase) throws InterruptedException {
+        TimeUnit.SECONDS.sleep(4);
+        this.fillFormController.fillLoginPage(this.webDriver,testCase);
+        this.fillFormController.clickLoginButton(this.webDriver);
+    }
+
+    @Then("I am not logged in FrontDesk")
```
Comment (luz-angelica) [inline src/test/java/com/gworks/stepDefinitions/ui/CityPageLogInStepDef.java]: chance this Then on third person please

File: src/test/java/com/gworks/stepDefinitions/ui/CitizenPageLogInStepDef.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    @When("The user uses a {string} credentials in Citizen page")
+    public void theUserUsesACredentialsInCitizenPage(String testCase) throws InterruptedException {
+        TimeUnit.SECONDS.sleep(4);
+        this.fillFormController.fillLoginPage(this.webDriver,testCase);
+        this.fillFormController.clickLoginButton(this.webDriver);
+    }
+
+    @Then("I am logged in Citizen")
```
Comment (luz-angelica) [inline src/test/java/com/gworks/stepDefinitions/ui/CitizenPageLogInStepDef.java]: chance this Then on third person please

File: src/test/java/com/gworks/stepDefinitions/ui/CitizenPageLogInStepDef.java [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    public void iAmLoggedInCitizen() throws InterruptedException {
+        TimeUnit.SECONDS.sleep(4);
+        Assert.assertEquals(this.webDriver.getTitle(),"Pay Bills - FrontDesk");
+    }
+
+    @Then("I am not logged in Citizen")
```
Comment (luz-angelica) [inline src/test/java/com/gworks/stepDefinitions/ui/CitizenPageLogInStepDef.java]: chance this Then on third person please

**Review by luz-angelica — APPROVED** [review]
Great Job!
Thank you 💯

**Pull request merged**